weeklyreport/README.md

```diff
@@ -1,26 +1,26 @@
 Weekly Report
 =================
 
-In the end of each week, both the mentor and the apprentice need to fill in a small report. The purpose of the report is to gather information to improve both the ongoing apprenticeship and the following ones.
+At the end of each week, both the mentor and the apprentice need to fill in a small report. The purpose of the report is to gather information to improve both the ongoing apprenticeship and the following ones.
 
 The main goals of the weekly report is to gather the following information.
 
-- [ ] Topics approached during the week.
-- [ ] Feedback from the mentor.
-- [ ] Feedback from the apprentice.
+* Topics covered during the week;
+* Feedback from the mentor;
+* Feedback from the apprentice.
 
-An online form is enough for the weekly report. The tool is optional, as long as it contains the necessary questions
+An online form is enough for the weekly report. The tool is optional, as long as it contains the necessary answers.
```

Comment:
Maybe changing "contains" to "provides", but the reasoning is that we don't care about the questions, the output that we need are the answers.